### PR TITLE
custom_com_hosted -> cartodb_com_hosted fixes #6536 and #6534

### LINKED ISSUE #6536: upgrade link goes to 404 error
Hi! I'm doing smoke tests and while doing "Check that upgrade link goes to upgrade" on the editor it sent me to this 404 page. 

screenshot 2016-02-09 10 37 44

### PR DESCRIPTION
**Pull request by juanignaciosl**
@xavijam , @javisantana , CR this, please. This fixes #6536 and #6534.

### CODE REVIEW

**Comment by xavijam** [comment]
For me :+1:

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def frontend_config
       datasource_search_twitter:  nil,
       max_asset_file_size:        Cartodb.config[:assets]["max_file_size"],
       watcher_ttl:                Cartodb.config[:watcher].try("fetch", 'ttl', 60),
-      upgrade_url:                cartodb_com_hosted? ? "''" : "#{current_user.upgrade_url(request.protocol)}",
+      upgrade_url:                cartodb_com_hosted? ? false : "#{current_user.upgrade_url(request.protocol)}",
```
Comment (xavijam) [inline app/helpers/application_helper.rb]: :joy:

**Comment by juanignaciosl** [comment]
Ok, @javisantana , @xavijam , CR this again, please (b7de506 change is small but not minor).

**Comment by xavijam** [comment]
Fine by me

**Comment by javisantana** [comment]
:es:

**Pull request merged**